core/src/main/java/hudson/triggers/SafeTimerTask.java

```diff
@@ -43,6 +50,14 @@
  * @since 1.124
  */
 public abstract class SafeTimerTask extends TimerTask {
+
+    /**
+     * System property to change the location where tasks logging should be sent.
+     * <p><strong>Beware: changing it while Jenkins is running gives no guarantee logs will be sent to the new location
+     * until it is restarted.</strong></p>
+     */
+    static final String TASKS_LOGS_ROOT_PATH_PROPERTY = SafeTimerTask.class.getName()+".TASKS_LOGS_ROOT_PATH";
```

Comment:
/me ponders: this sysprop name is not very consistent with `getLogsRoot()` intent, though it's the case for now, it's not only for **tasks** logging...

Reply:
Yes ".logsTargetDir" would be better. Maybe it also makes sense to move the property to another class. "SafeTimerTask" barely has any relation to logging for everybody excepting Jenkins core devs.

Reply:
@oleg-nenashev fixed. Didn't move to another class though, since I'm not sure which one to use and hence that it would really be better. Thanks again for the review!